event_registration_partner_unique/models/event.py

```diff
@@ -26,24 +26,23 @@ def _check_forbid_duplicates(self):
 class EventRegistration(models.Model):
     _inherit = "event.registration"
 
-    @api.multi
     @api.constrains("event_id", "attendee_partner_id")
     def _check_forbid_duplicates(self):
-        """Ensure no duplicated attendees are found in the event."""
-        for s in self.filtered("event_id.forbid_duplicates"):
-            dupes = self.search(s._duplicate_search_domain())
+        # Ensure no duplicated attendees are found in the event
+        for event_reg_id in self.filtered("event_id.forbid_duplicates"):
```

Comment:
Variable is not an ID, so the name is misleading

```suggestion
        for event_reg in self.filtered("event_id.forbid_duplicates"):
```